src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -77,7 +87,8 @@ private Loan(JsonObject representation, Item item, User user, User proxy,
 
     this.originalDueDate = originalDueDate == null ? getDueDate() : originalDueDate;
 
-    this.loanPolicy = loanPolicy;
+    this.overdueFinePolicyId = getProperty(representation, OVERDUE_FINE_POLICY_ID);
+    this.lostItemPolicyId = getProperty(representation, LOST_ITEM_POLICY_ID);
```

Comment:
I thought that part of the intention of using the Policies and Policy classes was so that this usage of the JSON representation of the loan could be moved outside of the Loan class and instead, any reference be based upon the respective policy class.

Did I not convey this intent effectively?

Reply:
Could you describe your approach in more detail? 
How can we move out the JSON representation of the loan outside of the Loan class if this representation contains overdueFinePolicyId and lostItemPolicyId?

Reply:
I can try.

An instance of the `Policies` class is passed to the `Loan` constructor. 

That instance is made up of various `Policy` class instances. Those can either be known, and have an ID and name, or unknown and only have an ID.

When the loan is created, the JSON could be interpreted, and unknown policies can be created for each one, using the ID from the JSON.

For example, the `loanPolicyId` would be used to create an `Unknown LoanPolicy` with that ID.

If the policies instance and each policy instance is always present, the loan can delegate to them in order to get the property.

Then there is no need for the loan to have fields for these values or use the JSON.

Does that make sense?

Reply:
Fixed